App/Sagas/TextileSagas.js

```diff
@@ -239,25 +239,61 @@ export function * shareImage ({thread, hash, caption}) {
 
 export function * photosTask () {
   try {
-    const photos = yield call(queryPhotos)
-    for (const photo of photos.reverse()) {
-      const multipartData = yield call(IPFS.addImageAtPath, photo.path, photo.thumbPath, 'default')
-      yield call(RNFS.unlink, photo.path)
-      yield call(RNFS.unlink, photo.thumbPath)
-      yield put(TextileActions.imageAdded('default', multipartData.boundary, multipartData.payloadPath))
-      yield put(IpfsNodeActions.getPhotoHashesRequest('default'))
-      yield call(
-        Upload.startUpload,
-        {
-          customUploadId: multipartData.boundary,
-          path: multipartData.payloadPath,
-          url: 'https://ipfs.textile.io/api/v0/add?wrap-with-directory=true',
-          method: 'POST',
-          type: 'multipart',
-          field: multipartData.boundary
+    const camera = yield select(TextileSelectors.camera)
+    const processed = camera && camera.processed ? camera.processed : []
+
+    let allPhotos = yield call(getAllPhotos)
+    if (processed.length === 0) {
```

Comment:
Thought of one edge case we can totally ignore for now if you want, buuuut... 

The way this works, if the user has 0 photos in the camera roll, installs and runs Textile, takes some photos, opens Textile, those photos will be added to the ignored list, and only after that will subsequent photos be processed.

Reply:
The fix would be for `camera.processed === null` to represent the state where the initial 'ignore photos' query has not yet been run. Once `camera.processed` exists, even if it is empty, there would be no need to query photos to ignore.

Reply:
okay, this edge case has occurred but thought so little hadn't thought of this solution. will try quickly here